internal/config/config.go

```diff
@@ -27,8 +27,9 @@ const (
 	CredentialsFileName    = "credentials.json"
 	DefaultPluginDir       = "plugins"
 
-	DefaultPostRetryDelaySec = 5
-	DefaultPostMaxRetries    = 20
+	DefaultPostRetryDelaySec  = 5
```

Comment:
@sanderblue where else are we using those values to retry less than 5min? If it's anywhere related to installation, we should probably update those values as well to retry up to 5 min.

Reply:
One other place we use retries is during the [`ConfigValidator.Validate()` method](https://github.com/newrelic/newrelic-cli/blob/main/internal/diagnose/config_validator.go#L72-L86) for the `postEvent` when calling `client.Events.CreateEventWithContext()`.

Do we also want to increase that timeout? Or are we only concerned with the license key fetch timeout?

Reply:
I think we should have a consistent retry/loop and consolidate on 5min max timeout. Otherwise, I'd be concerned we would have other flows that will experience a faster timeout when the account is too new.

Reply:
One thing to note, if we do increase the timeouts for all of these retry methods, it could increase the total installation time fairly dramatically if the timeouts are hit since we call these methods multiple times throughout the installation process.